### PR TITLE
fix pending_normal_aio_*

### PR DESCRIPTION
**Pull request by yseto**
Added the ability to parse values that could not be parsed before.

### CODE REVIEW

File: mackerel-plugin-mysql/lib/mysql.go [hunk 1]
```diff
@@ -773,6 +773,29 @@ func setIfEmpty(p map[string]float64, key string, val float64) {
 	}
 }
 
+func calcurateAio(s string) (int, error) {
+	v := strings.TrimSpace(s)
+	i := strings.Index(v, "[")
+	j := strings.Index(v, "]")
+
+	if i >= 0 && j >= i {
```
Comment (susisu) [inline mackerel-plugin-mysql/lib/mysql.go:781]: it's trivial though
```suggestion
	if i >= 0 && j > i {
```

**Review by susisu — COMMENTED** [review]
Looks good! I commented some minor points.

File: mackerel-plugin-mysql/lib/mysql.go [hunk 2]
```diff
@@ -773,6 +773,29 @@ func setIfEmpty(p map[string]float64, key string, val float64) {
...
+				return 0, err
+			}
+			total += p
+		}
+		return total, nil
+	}
+	if n := len(v); v[n-1] == ',' {
+		v = v[:n-1]
+	}
+	return strconv.Atoi(v)
```
[the remark this answers is not in the document]
Comment (yseto) [reply mackerel-plugin-mysql/lib/mysql.go:796]: Perhaps not needed, I think it is better to match the code expression in the sense of unifying, so I will change it

**Review by susisu — APPROVED** [review]
LGTM!

**Comment by yseto** [comment]
Thanks for your review.

**Pull request merged**